app/views/organisations/_show.html.erb

```diff
@@ -35,15 +35,27 @@
       <ul class="org-categories">
         <li id="what_they_do">
           <h5><strong>What they do</strong></h5>
-          <p><%= org.categories.what_they_do.pluck(:name).join(', ') %></p>
+          <p>
+          <% org.categories.what_they_do.collect do |category| %>
+          <%= link_to category.name, organisations_search_path(what_id: category.id, who_id: '', how_id: '', q: '') %>
+          <% end.join(', ') %>
+          </p>
         </li>
         <li id="who_they_help">
           <h5><strong>Who they help</strong></h5>
-          <p><%= org.categories.who_they_help.pluck(:name).join(', ') %></p>
+          <p>
+          <% org.categories.who_they_help.collect do |category| %>
+          <%= link_to category.name, organisations_search_path(what_id: '', who_id: category.id, how_id: '', q: '') %>
+          <% end.join(', ') %>
+          </p>
         </li>
         <li id="how_they_help">
           <h5><strong>How they help</strong></h5>
-          <p><%= org.categories.how_they_help.pluck(:name).join(', ') %></p>
+          <p>
+          <% org.categories.how_they_help.collect do |category| %>
+          <%= link_to category.name, organisations_search_path(what_id: '', who_id: '', how_id: category.id, q: '') %>
+          <% end.join(', ') %>
+          </p>
```

Comment:
I'm not sure what kind of helper you see fit here. I created something, though I have now problems with adding comas (`join(', ')` is not working).